### DISCUSSION TITLE
Module '"@headlessui/react"' has no exported member 'Tab'.

### DISCUSSION THREAD

**Discussion by w3nder** [tailwindlabs/headlessui#710] [Help]
Using version 1.4.0 is following the example in the documentation.
`import { Tab } from '@headlessui/react';`

**Comment by RamK777-stack** [comment]
I also facing this issue. @w3nder  did you found any solution ?.

**Comment by princeajuzie7** [comment]
uninstall the package and install it back will resolv the issue :)
